Tests/MutationSteps/BuildForTestingTests.swift

```diff
@@ -137,6 +137,26 @@ final class BuildForTestingTests: MuterTestCase {
             .literal(reason: "Could not parse xctestrun at path: some/project.xctestrun")
         )
     }
+    
+    func test_givenTestArgumentsContainsDerivedDataPath_shouldCopyXCTestrunCorrectly() async throws {
```

Comment:
Typo, but also, is this what you want to call it? Or is it more like `test_supportsCustomDerivedDataPath` or something?

```suggestion
    func test_givenTestArgumentsContainsDerivedDataPath_shouldCopyXCTestRunCorrectly() async throws {
```